gaphor/ui/elementeditor.py

```diff
@@ -249,6 +249,10 @@ def tx_update_style_sheet(style_sheet, text):
     def open(self, builder):
         self.style_sheet_buffer = builder.get_object("style-sheet-buffer")
         self.style_sheet_view = builder.get_object("style-sheet-view")
+        self.style_sheet_view.set_auto_indent(True)
+        self.style_sheet_view.set_indent_width(2)
+        self.style_sheet_view.set_input_hints(Gtk.InputHints.NO_EMOJI)
+        self.style_sheet_view.set_insert_spaces_instead_of_tabs(True)
```

Comment:
`style_sheet_view` is a `Gtk.TextView` and the 3 indention methods you added are for the `Gtk.Source.View`.